mappings/net/minecraft/network/NotMainThreadException.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS qn net/minecraft/network/NotMainThreadException
```

Comment:
`NotMain` -> `Off`?

Reply:
I agree that's better. I'm on my phone right now so I can't make any changes for around 4 hours, but maybe @asie can change it?